### ISSUE TITLE
Deprecation warning about DataFrame index on "Timeseries_Data_analysis.ipynb"

### ISSUE THREAD

**Issue by guillempuche** [srivatsan88/End-to-End-Time-Series#5]
Warning:
![imatge](https://user-images.githubusercontent.com/3519924/114300134-04822080-9abf-11eb-8eec-51981e681053.png)

You have to change lines:
- `aq_df_2016=aq_df['2016']` for `aq_df_2016=aq_df.loc['2016']`
- `df_2014= aq_df['2014'].reset_index()` for `df_2014= aq_df.loc['2014'].reset_index()`
- `df_2015 = aq_df['2015'].reset_index()` for `df_2015 = aq_df.loc['2015'].reset_index()`
- On several lines `aq_df_2015=aq_df['2015']` for `aq_df_2015=aq_df.loc['2015']`

I'm using:
- pandas `1.2.3`
- python `3.8.8`

**Comment by srivatsan88** [comment]
Thanks will make the changes and I modified my code and ran it on fit instead of fit_generator with TF2.5 and it works

early_stopping = tf.keras.callbacks.EarlyStopping(monitor='val_loss',
                                                    patience=2,
                                                    mode='min')

model.compile(loss=tf.losses.MeanSquaredError(),
              optimizer=tf.optimizers.Adam(),
              metrics=[tf.metrics.MeanAbsoluteError()])

history = model.fit(train_generator, epochs=20,
                    validation_data=val_generator,
                    shuffle=False,
                    callbacks=[early_stopping])

Will upload it once I add additional details to it
